src/user_interface/main_page.py

```diff
@@ -12,6 +10,55 @@ class MainPage(QWidget, Thread):
     def __init__(self, shutdown):
         super().__init__()
         Thread.__init__(self)
+        style = """
+            QWidget {
+                background: #2a0b40;
+            }
+            QLabel{
+                color: #fff;
+                font: 40px;
+            }
+            QPushButton{
+                color: #fff;
+                background-color: #5b2185;
+                border-style: outset;
+                border-width: 2px;
+                border-color: #792cb0;
+                max-width: 50em;
+                min-width: 5em;
+                padding: 5px;
+                font-family: "Times New Roman", Times, serif;
+                font: bold 15px;
+                border-radius: 10px;
+            }
+            QPushButton:hover{
+                background: #792cb0;
+            }
+            QPushButton:pressed{
+                border-style: inset;
+            }
+            QToolButton{
+                color: #fff;
+                background-color: #5b2185;
+                border-style: outset;
+                border-width: 2px;
+                border-color: #792cb0;
+                max-width: 25;
+                min-width: 5em;
+                padding: 5px;
+                font-family: "Times New Roman", Times, serif;
+                font: bold 15px;
+                border-radius: 10px;
+            }
+            QToolButton:hover{
+                background: #792cb0;
+            }
+            QToolButton:pressed{
+                border-style: inset;
+            }
+        """
+        qApp.setStyleSheet(style)
+
```

Comment:
This had to moved here because __main__ doesn't run when running from command line